accepted/2020/minimum-os-version/minimum-os-version.md

```diff
@@ -0,0 +1,245 @@
+# .NET 5 OS API Versioning
+
+# Introduction
+
+The format for [.NET 5 target framework monikers (TFMs)](https://github.com/dotnet/designs/pull/92) includes a version component that represents a revision of the operating system (OS) binding APIs, for example `net5.0-ios15.0` would include a binding to the iOS 15.0 APIs. However, it is a common pattern for apps and libraries to support running on older versions of the OS by using runtime checks to guard their calls so that they do not call APIs that are not available. This applies today for the Xamarin platforms (iOS, Android, Mac etc) and for UWP.
```

Comment:
```suggestion
The format for [.NET 5 target framework monikers (TFMs)](https://github.com/dotnet/designs/blob/master/accepted/2020/net5/net5.md) includes a version component that represents a revision of the operating system (OS) binding APIs. For example, `net5.0-ios15.0` would include a binding to the iOS 15.0 APIs. However, it is a common pattern for apps and libraries to support running on older versions of an OS by using runtime checks to guard their calls so that they do not call APIs that are not available. This applies today for the Xamarin platforms (iOS, Android, Mac etc) and for UWP.
```